.eslintrc.yml

```diff
@@ -0,0 +1,43 @@
+---
+ignorePatterns:
+  - '*.d.ts'
+  - 'dist'
+  - "rollup.config.ts"
+
+env:
+  browser: true
+
+extends:
+  - plugin:prettier/recommended
+
+parser: '@typescript-eslint/parser'
+
+parserOptions:
+  project: './tsconfig.json'
+  sourceType: module
+
+settings:
+  import/internal-regex: ^@ramp-instant/
+
+plugins:
+  - eslint-plugin-import
+  - simple-import-sort
+  - eslint-plugin-prettier
+
+rules:
```

Comment:
As I fan of single quotes I would recommend you to use single quotes for imports, strings etc :D

Reply:
Also ramp-instant uses single quotes mostly